CONFIGDOC.md

```diff
@@ -0,0 +1,94 @@
+## Configure Plugin for Freestyle Project

```

Comment:
Please add links to the beginning of this document. Each link should help the user navigate to the desired section without the need to go through the whole document.